Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -450,8 +456,14 @@ def prepare_enforcement_boundary_create_output(response: Dict) -> str:
     }
 
     headers = list(hr_outputs.keys())
+
+    if already_exists:
```

Comment:
Do we need this in the HR output? Can we just add a log line instead that notes an object with the name already exists?

Reply:
I think we should keep this message in the HR output as well so it would be clear to the user that the existing object has been used.

Reply:
What's the difference for the user? I think we should just log a message to the war room that the object already exists (assuming this is possible) and leave it out of the HR outputs.

Reply:
Yes, we can skip the HR part and I've updated the code to just log the message of an existing object. But the log message will not be displayed in the war room. instead it will be shown in the log files.